15/umbraco-cms/tutorials/inspiration/migrating-macros/migrating-macros.md

```diff
@@ -0,0 +1,891 @@
+---
+description: >-
+  Get started with developing a custom migration path for Macros to Blocks in the Rich Text Editors.
+---
+
+# Introduction
+As with many things in Umbraco, there are a multitude of ways on how to migrate away from macros and use the blocks in the RTE instead. In this article we will be showcasing a solution that lets you scan and then fix each macro one by one (or in batch). This solution will most likely not be used in a production setup, but we hope it can serve as an inspiration on how you can build your own solution specific your current setup. At the end of the article we will discuss a few other ways of running a larger migration.
+
+In this article we will be doing a one on one conversion from macro to block with each paramater matching the same named properties on an element document type. We are also keeping it relatively simple and are using pure text as values. If your migration deals with more complex types, we advice you to create an instance of the new data format and compare the old value against the new as there might have been more differences between the paramater type on the macro and the propertyEditor/datatype on the element document type.
+
+Because most people will be dealing with this kind of migrations when they move between LTS version from 13 to 15, we chose to do just that in this article. More precisly from 13.7.2 to 15.2.3
```

Comment:
Could we mention something in this line:

"Notice this example would also be compatible if you where to migrate from 13 to 17".

In this way a future read do not think they have to go to the v.15 version, but they need to understand that they can go from LTS to LTS. :-)